### ISSUE TITLE
The domain alias must be shown on the "DNS Records" page as a normal record

### ISSUE THREAD

**Issue by iliajie** [virtualmin/virtualmin-gpl#916]
Hello Jamie!

Since alias domains don’t have their own “DNS Records” page like top-level and sub-servers do, it would make sense to always display the alias DNS records on the “DNS Records” page, without needing to click the “Show all records” button.

**Comment by jcameron** [comment]
No I don't think so, since the alias domain DNS records are always copied from the target domain

**Comment by iliajie** [comment]
From a coding standpoint, I get it. But looking at it from a user’s view, it’s pretty confusing. There’s a separate UI page for adding a domain alias. We add it, then head over to the “DNS Records” page to check, but nothing shows up. Then we jump over to Cloudflare and see the DNS record for the alias there. After that, we go back to the UI and realize the record is hidden behind an extra button.

**Comment by jcameron** [comment]
I think that's still the right option, because we don't want users to have to see or manage alias domain DNS records.

**Comment by iliajie** [comment]
But when the admin enables the DNS feature for an alias, they can’t see the DNS record, whether the alias or the parent domain is selected. This is confusing. I don’t see why we wouldn’t want the admin to see the DNS record that was created.

**Comment by jcameron** [comment]
Maybe we could make the DNS Records page visible in read-only mode when the alias domain is selected? That would be pretty simple..

**Comment by iliajie** [comment]
> Maybe we could make the DNS Records page visible in read-only mode when the alias domain is selected? That would be pretty simple..

Yes, that could work. Though, is it difficult to make it clearly visible for the parent domain?

**Comment by jcameron** [comment]
Yes, I've just implemented this

**Comment by iliajie** [comment]
Thanks, Jamie! However, the links are still displayed, and it’s still possible to edit the records, e.g.:

<img width="846" alt="image" src="https://github.com/user-attachments/assets/fab73ec2-75a6-4db6-b035-f025c0dc3ea4">

**Comment by jcameron** [comment]
Ooops I forgot to commit another patch .. should be done now

**Comment by iliajie** [comment]
Now it's perfect! Thank you!
